### ISSUE TITLE
[Enhancement] Wallpaper preview selector

### ISSUE THREAD

**Issue by gios2** [PawCorp/walltaker#43]
It would be useful to be able to set in the link settings how the preview of the wallpaper should be shown, for users who for example use the monitor vertically, in order to show the correct preview.
As previews there should be at least: smartphone, tablet portrait, tablet landscape, vertical monitor, horizontal monitor

**Comment by pupgray** [comment]
Makes sense! I imagine this as NOT a restriction on the wallpapers that can be posted (blocking if wrong ratio) since sometimes the cropping is desirable (a wide picture could crop well on a vertical phone)

This would just be for the preview area.

**Comment by pupgray** [comment]
For anyone trying this as a first issue:

This would require a new migration to add a "preview_type" (suggested name) column to the links table. I'd suggest making a bigint, so you can use [rail's enum in the model.](https://guides.rubyonrails.org/active_record_querying.html#enums) This will let you instead use it with symbols like :smartphone_portrait to interact with this column.

Then, inside the [app/views/links/show.html.erb](https://github.com/PawCorp/walltaker/blob/main/app/views/links/show.html.erb) view, depending on the value of this column for a given link, display a cropped preview approximating the ratio of hight and width of a device
